mri/operators/fourier/non_cartesian.py

```diff
@@ -34,6 +34,13 @@
                   "the master release. Till then you cannot use NUFFT on GPU")
     pass
 
+try:
+    from gpuNUFFT import NUFFTOp
+except ImportError:
+    warnings.warn("gpuNUFFT python package has not been found. If needed use "
+                  "the master release.")
+    pass
```

Comment:
why this `pass`?

Reply:
To ensure that not having gpuNUFFT library still allows the use of pysap-mri . The user can use CPU NFFT still

Reply:
well you don't need a `pass` for that do you? the `except` should be enough

Reply:
Agreed, will change that